web/styles/right_sidebar.css

```diff
@@ -341,7 +341,11 @@ $user_status_emoji_width: 24px;
     display: flex;
     flex-direction: row;
     justify-content: space-between;
-    margin-top: 5px;
+    /* The margin top is calculated from a legacy 25px height,
+       from a 20px line of text and 5px of margin top. We calculate
+       a scaling margin-top by subtracting the em-unit line height
+       from the legacy value. */
+    margin-top: calc(25px - (var(--legacy-body-line-height-unitless) * 1em));
```

Comment:
Should this 25px also be a variable? Presumably it corresponds to the height of some other item this is supposed to align with.

Reply:
Honestly I derived that value based on the state of things in `main`, based on what I wrote in the CSS comment here. We could attach a variable name to it, but it appears to be an eyeballed value (the former `margin-top: 5px`) overall to me--just enough to place the element reasonably within the viewport in the right sidebar.

A better but more involved refactor would be to have this and the analogous item in the left sidebar share similar if not identical approaches to their positioning.

Reply:
OK let's just merge this and move on.